features/utils.js

```diff
@@ -21,10 +21,14 @@ const authHeader = new Buffer.from(
 ).toString('base64')
 
 const patientId = '12305759751'
-const patientId2 = '1234567'
+const patientId1 = '1234567'
+const patientId2 = '12305759781'
 const programId = 'uYjxkTbwRNf'
-let trackedEntityId, trackedEntityId2, fhirBundleCaseReport, fhirBundleLabResult, organizationsBundle, locationExists
+let trackedEntityId, trackedEntityId1, trackedEntityId2, fhirBundleCaseReport, fhirBundleCaseOutcome, fhirBundleLabResult, organizationsBundle, locationExists
 
+const caseOutcomeQuestionnaireResponse = JSON.parse(
+  fs.readFileSync(`${__dirname}/resources/case-outcome-questionnaireResponse.json`, 'utf8')
+)
```

Comment:
VS Code highlight your __dirname string concatenation as an issue. `path.resolve()` is preferable as it will work out the slashes for you. (This may make it easier to run on windows if we ever have to - hopefully not)
```suggestion
const caseOutcomeQuestionnaireResponse = JSON.parse(
  fs.readFileSync(path.resolve(__dirname, 'resources', 'case-outcome-questionnaireResponse.json'), 'utf8')
)
```